.github/pages/hello-world.md

```diff
@@ -35,6 +35,39 @@ This downloads the Fashion-MNIST training and test dataset and tells Jina to ind
 Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results.
 The whole process takes about 1 minute.
 
+<br><br>
+
+#### Use jina hub Executors
+
+
+ You can run the `jina hello fashion` demo using a different embedding method.
+ As an example, you can use  [ImageTorchEncoder](https://github.com/jina-ai/executor-image-torch-encoder)
+  changing in `jina/helloworld/fashion/app.py` the flow
+
+ ```python
+f = (
+     Flow()
+     .add(uses=MyEncoder, parallel=2)
+     .add(uses=MyIndexer, workspace=args.workdir)
+     .add(uses=MyEvaluator)
+     )
+ ```
+
+ with the  flow
+
+ ```python
+ f = (
+     Flow()
+     .add(uses='jinahub+docker://ImageTorchEncoder',
+          override_with={'model_name': 'alexnet'},
+          parallel=1)
```

Comment:
Can we use the default setting so that we won't freak out the beginners?
```suggestion
      )
```

Reply:
Ok, parallelization should be 1 until we bump the version to 2.0.5

Reply:
The issue with the default model is that the example will take a lot to execute because the default model is much bigger, so I think it makes sense to use an smaller model for this.